day_two_guide/topics/proc_bringing-openshift-online.adoc

```diff
@@ -0,0 +1,40 @@
+////
+bringing OpenShift back online
+
+Module included in the following assemblies:
+
+* admin_guide/assembly_restoring-cluster.adoc
+* upgrading/downgrade.adoc
+
+////
+
+[id='bringing-openshift-services-back-online_{context}']
+= Bringing {product-title} services back online
+
+. On each {product-title} master, restore your master and node configuration from
+backup and enable and restart all relevant services.
+
```

Comment:
Change the full stop above to a colon? And also put a + inbetween the para and example below? (This is rendering as two step 1s)